garrysmod/gamemodes/terrortown/gamemode/player_ext_shd.lua

```diff
@@ -182,7 +182,7 @@ if CLIENT then
    };
 
    -- Insert all the "simple" gestures that do not need weight control
-   for _, a in pairs{ACT_GMOD_GESTURE_AGREE, ACT_GMOD_GESTURE_DISAGREE,
+   for _, a in ipairs{ACT_GMOD_GESTURE_AGREE, ACT_GMOD_GESTURE_DISAGREE,

```

Comment:
lord

Reply:
in this place, the local table will only take up memory, this action takes place once in 1 start of the file.